src/generic_core/uproxy_core.ts

```diff
@@ -655,15 +655,64 @@ export class uProxyCore implements uproxy_core_api.CoreApi {
       };
       const MAX_INSTALLS = 5;
       return retry(install, MAX_INSTALLS);
-    }).then((output: string) => {
+    }).then((cloudNetworkData :any) => {
       destroyModules();
-      return <uproxy_core_api.CloudInstallResult>{
-        invite: output
-      };
+
+      // Login to the Cloud network if the user isn't already so that
+      // we can add the new cloud server to the roster.
+      return this.loginIfNeeded_('Cloud').then((cloudNetwork) => {
+        // Set flag so Cloud social provider knows this invite is for the admin
+        // user, who just created the server.
+        cloudNetworkData['isAdmin'] = true;
+
+        // Synthesize an invite token based on cloudNetworkData.
+        // CONSIDER: if we had a social.acceptInvitation that only took
```

Comment:
Ah, in this case social.acceptInvitation is referring to the social.ts file inside uProxy/uProxy - that method currently takes all the fields (including ```v```, ```networkName```, etc)